test/grizzly/inclusions_test.exs

```diff
@@ -40,24 +40,26 @@ defmodule Grizzly.InclusionsTest do
     end
   end
 
-  @tag skip: "Flaky"
   test "remove a node" do
-    :ok = Inclusions.remove_node()
+    :ok = Inclusions.remove_node(controller_id: 2000)
 
+    Process.sleep(10)
```

Comment:
How necessary are the `Process.sleep` calls?

I know how difficult it can be to get unit tests like this to run deterministically without some `sleep` calls, so it's not something I'd harp on; just checking!